app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,96 @@
  *= require_tree .
  *= require_self
  */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: rgb(235, 238, 238);
+}
+
+.d-flex {
+  display: flex;
+}
+
+.d-flex-column {
+  flex-direction: column;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.nav-bar-container {
+  font-family: 'Italiana', serif;
+  font-size: 1.2rem;
+  font-weight: bold;
+  justify-content: space-between;
+  padding: 1.5rem;
+  background-color: #3778c2;
+  color: #fff;
+}
+
+.sign-out-btn {
+  border: none;
+  background-color: transparent;
+  color: #fff;
+  font-size: 1.1rem;
+}
+
+.form-field {
+  width: 100%;
+  padding: 1rem;
+  font-size: 1rem;
+  font-family: 'Roboto', sans-serif;
+  outline: none;
+}
+
+.back-link {
+  color: white;
+}
+
+.devise-btn {
+  margin: 0 2rem;
+  width: 20rem;
+  text-align: center;
+  padding: 1rem;
+  font-family: 'Italiana', serif;
+  font-size: 1.5rem;
```

Comment:
- [ ] Here as well. I would recommend using the same font which is recommended in the[ design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).

Reply:
Thank you for the feedback, I have implemented this change 👍